src/daemon/daemon.cpp

```diff
@@ -1680,6 +1680,26 @@ try // clang-format on
             delayed_shutdown_instances.erase(name);
         }
 
+        try
+        {
+            auto& sshfs_mounts = mount_threads.at(name);
+            for (const auto& sshfs_mount : sshfs_mounts)
+            {
+                mpl::log(mpl::Level::debug, "delete",
+                         fmt::format("Stopping mount '{}' in instance \"{}\"", sshfs_mount.first, name));
+                sshfs_mount.second->stop();
+            }
+        }
+        catch (const std::out_of_range&)
+        {
+            mpl::log(mpl::Level::debug, "delete", fmt::format("No mounts to stop for instance \"{}\"", name));
+        }
+        catch (const std::exception& e)
```

Comment:
I wonder if this catch is really useful (since the whole function is wrapped in a try-catch). Do you think it would be harder to pinpoint the origin of an exception arising here if it was only caught below?

Reply:
I suppose letting the general catch below to the work would be roughly the same except that the delete operation would fail if a mount can't be stopped for whatever reason.

Reply:
Ok, after discussion on IRC and thinking about it some, the only reasons why we would get into this catch block would be for very strange reasons and would be worthy of failing a delete.  That said, I removed this:)